scripts/module/debug.nut

```diff
@@ -10,10 +10,10 @@ function onPlayerCommand( player, cmd, text ) {
 
     collectgarbage();
     ReloadScripts();
-  } else if (cmd == "pos") {
+  } else if (cmd == "get_pos") {
     print(player.Pos);
   } else if (cmd == "help") {
-    MessagePlayer("[#8080A0]/r[eload] /pos", player);
+    MessagePlayer("[#8080A0]/r[eload] /get_pos", player);
```

Comment:
I wouldn't bother with renaming this one, as there's no analogous `/set_pos` anymore.